bin/test/i18n/resources/dashboard/course_offerings/test_sync_out.rb

```diff
@@ -33,6 +33,8 @@
     end
 
     before do
+      I18n::Metrics.stubs(:report_runtime).yields(nil)
+
```

Comment:
I understand that we want to stub `report_runtime` so it doesn't log metrics in CloudWatch.